README.md

```diff
@@ -77,15 +76,15 @@ To build with the smaller C libraries, modify the application configuration file
 }
 ```
 
-If the build system throws an error when compiling with the small C library, you can find more information [here]( https://github.com/ARMmbed/mbed-os-5-docs/blob/development/docs/program-setup/bare_metal/c_small_libs.md).
+The build system build system reverts to the standard C library if support for the small C library is not enabled for your target. You can find more information [here]( https://github.com/ARMmbed/mbed-os-5-docs/blob/development/docs/program-setup/bare_metal/c_small_libs.md).
```

Comment:
Could you link to https://os.mbed.com/docs/mbed-os/v6.0-preview/bare-metal/using-small-c-libraries.html
@iriark01 what brnach should we use?

Reply:
For now use v6.0-preview, as /latest/ doesn't redirect to it (because it's not official, so /latest/ goes to 5.15)

Reply:
but we don't really want to update the link...

Reply:
Perhaps better to leave the link to github then